source/read-write-concern/read-write-concern.rst

```diff
@@ -301,11 +301,13 @@ Other commands that write
 
 Command helper methods for commands that write, other than those discussed above,
 MAY accept a write concern or write concern options in their parameter list.
+If the helper accepts a write concern, the driver MUST error if the selected server's MaxWireVersion < 5 and a
+write concern has explicitly been specified.
```

Comment:
I'm coming back to this now, I think we're ok to merge this specific change for 'Other commands that write', right?

As for findAndModify, Craig said he thinks it's fine to indicate to throw an error for < 3.2 so I'll add that text in.